l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -53,6 +53,10 @@ def _auditfile_name_get(self):
     date_generated = fields.Datetime(
         'Date generated', readonly=True, copy=False)
     company_id = fields.Many2one('res.company', 'Company', required=True)
+    account_account_ids = fields.Many2many('account.account',
```

Comment:
wouldn't you expect such a list to be an *restriction* list instead of *exclude*? try to make variable names intuitive, so I'd suggest `exclude_account_ids` (with repetitive model names we often skip one of then, also with `project.project` and the like; many people including me make the difference with analytic accounts that those are `{,*_}analytic_account_id{,s}`)